BuiltInSpicyAnalyzer.cmake

```diff
@@ -78,6 +78,12 @@ function (spicy_add_analyzer)
         if (SPICY_ROOT_DIR)
             target_include_directories(${lib} PRIVATE ${SPICY_ROOT_DIR}/include)
         endif ()
+
+        # Install Spicy grammars.
+        set(_SPIYC_SOURCES ${SPICY_ANALYZER_SOURCES})
+        list(FILTER _SPIYC_SOURCES INCLUDE REGEX "\.spicy$")
+        install(FILES ${_SPIYC_SOURCES} DESTINATION include/grammars/${NAME_LOWER})
```

Comment:
cmake noob: Would that install files into `<prefix>/include/grammars/`? That is, where all C++ headers are?

There is `<prefix>/share/spicy/` or `<prefix>/share/zeek/spicy/` and seems `grammars/` might fit quite well in the latter or maybe former?
```
$ tree share/zeek/spicy/
share/zeek/spicy/
├── zeek_file.spicy
├── zeek_rt.hlt
└── zeek.spicy
```

Reply:
Yes, that's exactly where they would end up. See my comment above on using `share/zeek/spicy` instead.